caffe2/python/onnx/backend.py

```diff
@@ -703,6 +711,10 @@ def make_gru(direction_offset):
             pred_mh.net.Concat([hidden_t_last_f, hidden_t_last_b],
                                [n.outputs[1], dummy_name()], axis=0)
 
+        if sequence_lens is not None:
+            pred_mh.net.VariableLengthSequencePadding(
```

Comment:
how is this going to consider drop_stats==False case where you have to pull them? Also is this covered by unit-tests?

Reply:
yes, it's covered by tests, see my top-level comment.

I don't understand what you mean about "drop_states == False case". drop_states is always False, and this works just fine.

Reply:
if we don't drop states, then we don't 0 fill, instead we pull the latest result to the end. On contrary if drop_states is True, then we need to do the padding logic. So in one case it is padding, in another case it is kind of gather.

Reply:
alex - this is onnx backend, i.e. we want to implement functionality of ops produced from other frameworks. So we can choose what drop_state we need to implement functionality and looks like we using False here. Anders - correct me if I'm wrong

Reply:
Yeah, I think I misread the code, didn't get used to the new github UI, sorry about that guys.